### ISSUE TITLE
New control: Background color in Main Menu Widget

### ISSUE THREAD

**Issue by fxaviers** [elementor/elementor#17838]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I would like a background color control to be added, for the menu widget, at the main level, in the normal state, the same as it already exists in hover and active.

Background color, in the advanced tab, is not the same thing, as it prints a color throughout the menu. What I would like is for each menu item to look like a button, with the possibility that the spacing between items does NOT have the color.

![image](https://user-images.githubusercontent.com/17536232/153949511-2500169f-ad72-4dba-aeef-f6406f6af245.png)

### Describe the solution you'd like

Add the background color control in the normal tab, as it already exists in active and hover.

**Comment by vgstef** [comment]
Yes, I don't understand why it isn't already there. We have background color for Hover and Active, but not for normal! I tried to set the background color in Custom CSS, but now the Hover and Active background color don't work anymore :(  Any solution?

**Comment by nicholaszein** [comment]
Hi,

This option is now possible by using the new Mega Menu widget. It's available as an experiment in the Elementor **`Settings`** > **`Features`** > **`Menu`** experiment. Just activate it and start building your menus directly from the Editor with much more granular control, and even adding the possibility to insert widgets to each menu content.

https://elementor.com/help/what-is-mega-menu/

This feature was added to Elementor Pro v3.12.0, back in 2023.

Thank you for sharing your feedback!

Kind regards
